### DISCUSSION TITLE
Ability to stop the selected container

### DISCUSSION THREAD

**Discussion by seba11** [louislam/dockge#245] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

When we have many of containers in our compose files only option to stop is all containers. Will be a good feture to have ability stop only selected container. 

### ✔️ Solution

Maybe a stop button near the bash like this ?

![Bez tytułu](https://github.com/louislam/dockge/assets/7965273/10016f1c-f0c8-4a55-b758-2004cbd61cc6)

**Comment by goodfore** [comment]
I would also like this feature. I would suggest add ability to stop/start, restart, and delete (remove) the container. Thanks!

**Reply by AgentRatty** [reply]
Delete CONTAINER!! I hit delete thinking there's no way it would delete the base folder, etc. only to be horrified that it did. For this reason I jump between portainer and Dockge. I like the detail view and ability to update/stop/start and edit yaml from dockge, but like the summary view and ability to delete the CONTAINER from portainer.

**Reply by Celer21** [reply]
Been there, done that, lost data 😄  https://github.com/louislam/dockge/discussions/405 
I hope, there will be option to delete only .env and compose.yaml file, not other files.

In the meantime, i have data in /docker/data and stack in /docker/stack.
Still i like Docgke more than portainer.

**Comment by qroac** [comment]
One more very useful option would be to update a single container.
So you do not need to restart the entire stack if just the service for the web frontend is updated.

**Reply by SP3NGL3R** [reply]
if only a single container has an update and you click "update", it does exactly as you described.

**Comment by Boburto** [comment]
I would also like this feature. Start/Stop/Update individual part of a stack.

**Comment by cbuxton5200** [comment]
I would really like this feature as well.  It would be equivalent to "docker compose restart myservice"

**Comment by wsw70** [comment]
related: https://github.com/louislam/dockge/discussions/123

**Comment by mizady** [comment]
pull request made for this feature, in the related discussion i have built a docker container of it if you don't want to fight with getting it to build yourself.  https://github.com/louislam/dockge/discussions/123#discussioncomment-10288371

https://github.com/louislam/dockge/pull/649
